iris-mpc-common/src/config/mod.rs

```diff
@@ -211,6 +212,121 @@ pub struct Config {
     pub hawk_server_reauths_enabled: bool,
 }
 
+impl Config {
+    pub fn common_config_hash(&self) -> String {
+        let mut hasher = sha2::Sha256::default();
+
+        // This is destructured here intentionally to cause a compile error if
+        // any of the fields are added to the struct without being considered if they should be in the common config hash or not.
+        let Config {
+            environment,
+            party_id: _,           // party id is different for each server
+            requests_queue_url: _, // requests queue url is different for each server
+            results_topic_arn,     // TODO: do we want the results topic ARN in the hash?
+            kms_key_arns: _,       // kms key arns are different for each server
+            service: _,
+            database: _,     // database is different for each server
+            cpu_database: _, // cpu database is different for each server
+            aws: _,          // aws is different for each server
+            processing_timeout_secs,
+            startup_sync_timeout_secs,
+            public_key_base_url: _, // TODO: include in common config hash?
+            shares_bucket_name: _,  // TODO: should this be the same for all servers?
+            clear_db_before_init,
+            init_db_size,
+            max_db_size,
+            max_batch_size,
+            heartbeat_interval_secs,
+            heartbeat_initial_retries,
+            fake_db_size,
+            return_partial_results,
+            disable_persistence,
+            enable_debug_timing: _,
+            node_hostnames, // TODO: is this the same for all servers?
```

Comment:
yes, all the same

Reply:
Except... When running locally. Let's leave it out for now

Reply:
See: https://github.com/worldcoin/iris-mpc/blob/36b5b4732ab93b767de254eb2b5b2858ca1f0035/.test.hawk0.env#L3

We'd have to change this, otherwise locally this will fail. Let's leave it out for now.

Reply:
Yes, this 0.0.0.0 is not necessary.